### ISSUE TITLE
Sold 2 OBJKTs but only received payment for 1

### ISSUE THREAD

**Issue by SolarKittyn** [hicetnunc2000/hicetnunc#1211]
**Describe the bug**
Today I sold 2 OBJKTs, and only received payment for 1. It was 1 person who purchased both of them and I can see the NFTs  in their collection so they did in fact receive them, but no funds have shown up in my temple wallet. I can see in the activity section of my wallet that i received 1 payment, but the one i'm missing is obviously not showing up there at all. I am unsure which one is missing of the 2 so i will include both OBJKT #s. The transaction in the screenshot I provided shows that they purchased 1 but not 2. Could it just be a bug of some kind?

**Please give us your profile link, tezos wallet address, and objkt numbers**

My Link- https://www.hicetnunc.xyz/solarkittyn/creations
My Wallet Address: tz1cL6BeFapdc87dd68Jj5ZaPUWsaX2UYzMN
OBJKTS #s- OBJKT#343020 & OBJKT#342999

>>This is the address it takes me to when i click on OBJKT under the OBJKT in the listings tab.
/The person who purchased: KT1Hkg5qeNhfwpKW4fXvq7HGZB9z2EnmCCA9

**Expected behavior**
I'm unsure why the funds arent showing up but i'm hoping it is just some sort of bug that can be fixed.

**Screenshots**
Provided

**Desktop (please complete the following information):**
 - OS: UBUNTU
 - Browser-  BRAVE BROWSER
 

**Additional context**
I provided a screenshot as well from the block explorer, and it does show 2 transactions to my wallet, but not of the right amount that the OBJKT sold for. It's way less. So it's showing on my account that my NFT sold for 7 or 8 TEZ but in the block explorer showing like 0.6. Very confusing!

![Screenshot from 2021-09-22 19-29-52](https://user-images.githubusercontent.com/91228683/134436102-af061b4f-caea-45b9-8f86-78efc63a706e.png)

![Screenshot from 2021-09-22 19-29-59](https://user-images.githubusercontent.com/91228683/134436120-208d9388-5f13-41d8-a5fa-879922a96f19.png)

![Screenshot from 2021-09-22 19-30-32](https://user-images.githubusercontent.com/91228683/134436135-0392385a-de67-49f2-b835-1c29c0626382.png)

![Screenshot from 2021-09-22 19-30-40](https://user-images.githubusercontent.com/91228683/134436143-95c7f487-0cdd-4f74-b4f9-5613abf8cf88.png)

![Screenshot from 2021-09-22 19-32-04](https://user-images.githubusercontent.com/91228683/134436154-a5a5eb16-8c8f-420a-93f0-ece4d3c8655e.png)

![Screenshot from 2021-09-22 19-32-11](https://user-images.githubusercontent.com/91228683/134436161-3a6c8b4c-72e1-4c5d-8387-b0383960d161.png)

![Screenshot from 2021-09-22 19-37-02](https://user-images.githubusercontent.com/91228683/134436174-fd419217-1946-4b04-a36f-7b25c6661740.png)

**Comment by violetforest** [comment]
Hey, please use this tool to see what you have sold 

https://hicdex.com/sold?address=tz1cL6BeFapdc87dd68Jj5ZaPUWsaX2UYzMN

The objkts that you linked to appear to be on the objktv1 contract. we had a problem on the site yesterday where everything that was swapped was swapped on the wrong contract. please try canceling the swaps and reswapping them.

**Comment by SolarKittyn** [comment]
According to the tool you provided, the 2 i wrote this post about is not showing up at all on there. Now i see that this morning i made another sell, and it went to their collection, and still no funds received so now im missing payment for 2 seperate ones. One for 7 tez and 1 for 8. This morning i still have the same amount of tez AS I did last night.

In the screenshot, it's showing my NFT in this persons collection but i have only have 1 payment showing up out of 3.
![Screenshot from 2021-09-23 12-44-57](https://user-images.githubusercontent.com/91228683/134549867-3fb3d8a7-9386-4061-a9a6-7277b3a6f3df.png)

**Comment by SolarKittyn** [comment]
What happens if i cancel a swap that someone supposedly purchased? I wouldnt want to cancel on someone if they did what they were supposed to on their end.

**Comment by violetforest** [comment]
https://www.hicetnunc.xyz/objkt/343020
https://www.hicetnunc.xyz/objkt/342999
both of these show up as "objktswap1" as the owner. this does not mean somebody bought it. this means you need to cancel it and reswap it.

when you swap something and put if for sale then the token is sent out of your wallet to the hicetnunc escrow wallet. when you cancel it it comes back to you.

the addresses that show in the screenshot in your wallet are the hicetnunc wallets
https://tzkt.io/KT1Hkg5qeNhfwpKW4fXvq7HGZB9z2EnmCCA9/
https://tzkt.io/KT1HbQepzV1nVGg8QVznG7z4RcHseD5kwqBn/operations/?utm_source=tezosnotifierbot

**Comment by SolarKittyn** [comment]
This one "Beautiful Stranger" was definitely sold to someone and it shows up on the tool you sent. underneath it shows someone by the name supernoob bought it, but i still have not received payment. the 2 i originally wrote this post about does not show up in the tool but Beautiful stranger does.

![Screenshot from 2021-09-23 14-13-56](https://user-images.githubusercontent.com/91228683/134561656-ff4508e9-6647-4685-9cc4-49795f934c6c.png)

**Comment by SolarKittyn** [comment]
![Screenshot from 2021-09-23 14-19-01](https://user-images.githubusercontent.com/91228683/134562190-519bf2d3-59bc-49ba-8477-d6f3f7cd17c4.png)

**Comment by violetforest** [comment]
The two that you originally posted about were not sold.

The payment for "Beautiful Stranger" shows in your screenshot. You sold the art for 6 tez, some fees went to Hicetnunc, and the rest went to you in two seperate transactions of 5.25 and 0.6 tez. the rest went to fees for HEN 

You can read more about the fees here https://github.com/hicetnunc2000/hicetnunc/wiki/Maintenance-fees

**Comment by SolarKittyn** [comment]
Ive recived  payment for 1 at 6 tez, but definitely sold 2. I sold Dahlia and Beautiful stranger. Dahlia is showing a buyer underneath.
![Uploading Screenshot from 2021-09-23 14-21-19.png…]()

**Comment by SolarKittyn** [comment]
I definitely understand about the 2 originally posted about.

**Comment by violetforest** [comment]
![transaction](https://user-images.githubusercontent.com/8921838/134562732-937c63df-f9f2-404d-9c84-b8a6f000664b.PNG)


if you scroll down you;ll see the 7 tez transaction for Dahlia broken up into two transactions

**Comment by SolarKittyn** [comment]
![Screenshot from 2021-09-23 14-21-19](https://user-images.githubusercontent.com/91228683/134562830-a9e0fbaf-a436-44c6-8cf6-e1ef9f2f6b76.png)

**Comment by SolarKittyn** [comment]
Ok, so the 2 payments showing up in my wallet are for Beautiful stranger and Dahlia?
And the other 2 i originally posted about aren't sold and need to be reswapped?

Thank you for your help.

**Comment by violetforest** [comment]
yes, I'm going to close this but let me know if you need anything else

https://github.com/hicetnunc2000/hicetnunc/wiki/How-to-swap-%F0%9F%94%83

**Comment by SolarKittyn** [comment]
(Sorry for the late response)

Ok, I believe I was able to get it all straightened out among having to pay the fees again for re-swapping. 
I understand it all now. I wasn't aware of the OBJKTV thing, so when I checked and seen they weren't for sale anymore and it was routing me to a collection account page, I assumed they had sold. I didn't take the escrow thing into account. Sorry for all the confusion... Learning process lol.
Thanks for being patient with me!
